engine/src/main/java/org/terasology/rendering/nui/widgets/UISlider.java

```diff
@@ -173,14 +171,39 @@ public String getMode() {
             return DISABLED_MODE;
         }
 
-        if (active) {
+        if (active || (TabbingManager.focusedWidget != null && TabbingManager.focusedWidget.equals(this))) {
```

Comment:
If you do `this.equals(TabbingManager.focusedWidget)` then you don't need the bulky null check here, as `equals` returns false for null